doc/reference/tooling/tt_cli/import.rst

```diff
@@ -134,14 +149,66 @@ To skip rows whose data cannot be parsed correctly, use the ``--on-error`` optio
     $ tt crud import localhost:3301 customers.csv:customers \
                      --on-error skip
 
+.. _tt-import-tdg2:
+
+Importing into Tarantool Data Grid 2
+------------------------------------
+
+To import data into TDG2, prepare a JSON lines file with an input data and call
+``tt tdg2 import`` using this file as an input:
+
+.. code-block:: console
+
+    $ tt tdg2 import localhost:3301 customers.jsonl:customers
+
+The input file can look like this:
+
+.. code-block:: json
+
+    {"age":30,"first_name":"Samantha","id":1,"second_name":"Carter"}
+    {"age":41,"first_name":"Fay","id":2,"second_name":"Rivers"}
+    {"age":74,"first_name":"Milo","id":4,"second_name":"Walters"}
+
+.. note::
+
+    Since JSON describes objects in maps with string keys, there is no way to
+    import a field value that is a map with a non-string key.
+
+If an error happens during TDG2 import, the all the changes made within the current batch
+are rolled back. The rollback process is the same as in ``tt crud import`` with the ``--rollback-on-error`` option.
+
+Batch rollback prevents the import of correct tuples that fall in a batch with errors in data,
```

Comment:
The batch rollback is not the reason for complication. It is
- the absence of cluster batch rollback,
- the absence of error matching.

Per-storage batch rollback makes this burden a bit easier, since at least something is rolled back when we report that "this batch has failed". Due to the absence of cluster batch rollback, it is likely that some part of the batch is actually applied. Due to the absence of error matching, we cannot say which part of the batch has been applied and which has been failed. We also cannot say which exact record caused the batch to fail.